ndcube/extra_coords/extra_coords.py

```diff
@@ -352,3 +357,21 @@ def __getitem__(self, item):
         # If we get here this object is empty, so just return an empty extra coords
         # This is done to simplify the slicing in NDCube
         return self
+
+    @property
+    def dropped_world_dimensions(self):
+        """
+        Return a APE-14 a-like representation of any sliced out world dimensions.
+        """
+
+        if self._wcs:
+            if isinstance(self._wcs, SlicedLowLevelWCS):
+                return self._wcs.dropped_world_dimensions
+
+        if self._lookup_tables or self._dropped_tables:
+            mtc = MultipleTableCoordinate(*[lt[1] for lt in self._lookup_tables])
```

Comment:
`self._lookup_tables` is a list of tuples which is `(index, table)` this is just pulling the tables out of the tuple.

We construct a `MultipleTableCoordinate` here to reduce duplication in functionality. `MultipleTableCoordinate` supports dropping dimensions when used standalone, by constructing it here we can use that implementation rather than re-implementing it. We don't keep one around all the time because we need to keep a note of what axes correspond to which tables.